Code/Framework/AzToolsFramework/AzToolsFramework/Input/QtEventToAzInputMapper.cpp

```diff
@@ -208,10 +214,15 @@ namespace AzToolsFramework
     void QtEventToAzInputMapper::SetEnabled(bool enabled)
     {
         m_enabled = enabled;
-        if (!enabled)
+        if (m_enabled)
+        {
+            AzFramework::InputChannelNotificationBus::Handler::BusConnect();
+        }
+        else
         {
             // Clear input channels to reset our input state if we're disabled.
             ClearInputChannels(nullptr);
+            AzFramework::InputChannelNotificationBus::Handler::BusDisconnect();
         }
     }
```

Comment:
Do we want to do all this if m_enabled == enabled? From the looks of it all this will be redundant (albeit benign) if so.

Reply:
Shouldn't hurt to add that guard in.